src/routes/(app)/pos/touch/+page.svelte

```diff
@@ -45,6 +45,55 @@
 	$: displayedProducts = productFiltered.slice(next, next + POS_PRODUCT_PAGINATION);
 	$: totalPages = Math.ceil(productFiltered.length / POS_PRODUCT_PAGINATION);
 	$: currentPage = Math.floor(next / POS_PRODUCT_PAGINATION) + 1;
+
+	async function removeLastItem() {
+		if (!items.length) {
+			return;
+		} else {
```

Comment:
The whole point of the early return pattern is there's no need for an `else` afterwards

```suggestion
		}
```